cpp/harvest/marc_issn_lookup.cc

```diff
@@ -395,6 +396,22 @@ void CleanDuplicationOfField773(MARC::Record * const record) {
     }
 }
 
+void CleanDuplicationOfField773ByTitle(MARC::Record * const record) {
+    std::vector<std::string> titles;
+    const ControlNumberGuesser control_number_guesser;
+    for (auto field(record->begin()); field != record->end(); ++field) {
+        if (field->getTag() == "773") {
+            const std::string title(control_number_guesser.NormaliseTitle(field->getFirstSubfieldWithCode('t')));
+            if ((std::find(titles.begin(), titles.end(), title)) != titles.end())
```

Comment:
Right now this logic is keeping the first 773 entry with the same title and throwing away the others. But in fact, we cannot rely on the first being the best, so we must check for the first occurrence with that title which actually has a w subfield, keep that one and remove all the others with the same title.